### PR TITLE
Changed every shortcut to use alt with the same key is was before.

### PR DESCRIPTION
**Pull request by taltaub22**
I did not want to make a lot of changes. So, I just checked to see if alt is also pressed.
I chose alt because ctrl+s / ctrl+a have affect on the page.

### CODE REVIEW

File: js/ClockCtrl.js [hunk 1]
```diff
@@ -285,20 +286,24 @@ angular.module('Clock').controller('ClockCtrl',[
                     }
                     break;
                 case 67:    //c
-                    $window.open('controls.html','fllClockControlWindow','resize=yes,width=600,height=800');
+                    $window.open('controls.html', 'fllClockControlWindow', 'resize=yes,width=600,height=800');
                     break;
-            };
+            }
+            ;
```
Comment (idanstark42) [inline js/ClockCtrl.js]: Please note these indentations

Comment (taltaub22) [reply js/ClockCtrl.js]: Woops

Comment (idanstark42) [reply js/ClockCtrl.js]: Do you want to fix it or to merge it like this

Comment (taltaub22) [reply js/ClockCtrl.js]: Yes, I do. Will fix it tonight

**Pull request merged**